alpha/apps/kaltura/modules/extwidget/actions/embedIframeAction.class.php

```diff
@@ -122,18 +122,20 @@ public function execute()
 			$uiconf_id = $widget->getUiConfId();
 		if(!$uiconf_id)
 			KExternalErrors::dieError(KExternalErrors::MISSING_PARAMETER, 'uiconf_id');
-			
+
 		$partner_host = myPartnerUtils::getHost($partner_id);
 		$partner_cdnHost = myPartnerUtils::getCdnHost($partner_id);
 
 		$uiConf = uiConfPeer::retrieveByPK($uiconf_id);
 		if(!$uiConf)
 			KExternalErrors::dieError(KExternalErrors::UI_CONF_NOT_FOUND);
-			
+
 		$partner_host = myPartnerUtils::getHost($partner_id);
 		$partner_cdnHost = myPartnerUtils::getCdnHost($partner_id);
-		
-		$html5_version = kConf::get('html5_version');
+
+		$embedPlaykitConf = kConf::getMap(kConfMapNames::APP_VERSIONS);
```

Comment:
Better approach in one line -
kConf::getArrayValue('html5_version','playerApps',kConfMapNames::APP_VERSIONS,null);

Signature - getArrayValue($paramName, $sectionName, $mapName = 'local', $defaultValue = false)